tgui/packages/tgui/interfaces/ReagentExtractor.js

```diff
@@ -0,0 +1,340 @@
+          <Flex.Item grow
+            overflow="hidden"
+            style={{
+              "text-overflow": "ellipsis",
+              "text-transform": "capitalize",
+            }}>
+            {container.name}
+          </Flex.Item>
+          <Flex.Item px={4} /> {/* this prevents the title buttons from being overlapped by the title text */}
+        </Flex>
+      }
+      buttons={
+        <>
+          <Button
+            tooltip="Flush All"
+            icon="times"
+            color="red"
+            disabled={!container.totalVolume}
+            onClick={() => act('flush', { container_id: container.id })}
+          />
+          {!insertable || (
+            <Button
+              tooltip="Eject"
+              icon="eject"
+              disabled={!props.container}
+              onClick={() => act('ejectcontainer')}
+            />
+          )}
+        </>
+      }>
+      {!!props.container || (
+        <Dimmer>
+          <Button
+            icon="eject"
+            fontSize={1.5}
+            onClick={() => act('insertcontainer')}
+            bold>
+            Insert Beaker
+          </Button>
+        </Dimmer>
+      )}
+      <ReagentGraph container={container} />
+      <ReagentList container={container} />
+      <Flex wrap justify="center">
+        <Flex.Item grow />
+        <Flex.Item grow>
+          <Button
+            mb={0.5}
+            width={17}
+            textAlign="center"
+            selected={container.selected}
+            tooltip="Select Extraction and Transfer Target"
+            icon={container.selected ? "check-square-o" : "square-o"}
+            onClick={() => act('extractto', { container_id: container.id })}
+          >
+            Select
+          </Button>
+        </Flex.Item>
+        <Flex.Item>
+          <Flex width={17}>
+            <Flex.Item grow>
+              <Button
+                disabled={container.selected}
+                onClick={() => act('chemtransfer', { container_id: container.id, amount: transferAmount })}
+              >
+                Transfer
+              </Button>
+              <NumberInput
+                value={transferAmount}
+                format={value => value + "u"}
+                minValue={1}
+                maxValue={500}
+                onDrag={(e, value) => setTransferAmount(value)}
+              />
+            </Flex.Item>
+            <Flex.Item>
+              <Button
+                disabled={container.selected}
+                onClick={() => act('chemtransfer', { container_id: container.id, amount: 500 })}
+              >
+                Transfer All
+              </Button>
+            </Flex.Item>
+          </Flex>
+        </Flex.Item>
+      </Flex>
+    </Section>
+  );
+};
+
+const ReagentGraph = (props, context) => {
+  const { container } = props;
+  const { maxVolume, totalVolume, finalColor } = container;
+  const contents = container.contents || [];
+
+  return (
+    <Fragment>
```

Comment:
I believe we're converting over to `<>` and `</>` instead of outright saying Fragment